edge/pkg/apis/dmi/v1/api.proto

```diff
@@ -0,0 +1,298 @@
+syntax = "proto3";
+
+//option go_package = "path;name";
+option go_package="./;v1";
+package v1;
+
+import "google/protobuf/any.proto";
+
+service DeviceManagerService {
+  rpc MapperRegister(MapperRegisterRequest) returns (MapperRegisterResponse) {}
+
+  rpc ReportDeviceStatus(ReportDeviceStatusRequest) returns (ReportDeviceStatusResponse) {}
+}
+
+message MapperRegisterRequest {
+  bool withData = 1;
```

Comment:
what does withData  mean?

Reply:
if withData is true, the device manager should return the device list to the mapper. if false, it means the mapper could get the device list locally and do not need to get it from device manager.